src/shared/containers/ResourceFormContainer.tsx

```diff
@@ -11,10 +11,8 @@ const ResourceFormContainer: React.FunctionComponent<{
 }> = ({ orgLabel, projectLabel }) => {
   const nexus = useNexusContext();
   const createResource = (schemaId: string, payload: ResourcePayload) => {
-    // CANNOT create resource with schemaID
-    // TODO: update after fix
-    // https://github.com/BlueBrain/nexus/issues/788
-    return nexus.Resource.create(orgLabel, projectLabel, payload);
+    // TODO: bump @bbp/nexus-sdk
```

Comment:
:thinking: 
sdk 1.1.8 was release with `schemaId` support https://github.com/BlueBrain/nexus-js/releases/tag/v1.1.8

Reply:
you're right 🙊